pinc/RoundDescriptor.inc

```diff
@@ -199,6 +199,25 @@ class Round extends Stage
         // Give them the lowest honorific.
         return $honorific;
     }
+
+    // -----------
+
+    public function user_can_access()
```

Comment:
Similar: `user_can()` should return an answer. `validate_user_can()` can return an exception as the answer.